misc/init.d/cfengine3.in

```diff
@@ -182,7 +182,7 @@ cf_get_matching_pids()
 
     if [ $INSIDE_CONTAINER = -1 ]; then
         # Are we inside a container?
-        if [ -f /proc/self/cgroup ] && egrep '[0-9]+:devices:/[^:]' /proc/self/cgroup > /dev/null; then
+        if [ -f /proc/$pid/cgroup ] && sed 's_/\(-\|system\|user\).slice__;s_/user-[0-9]*.slice__;s_/session-[0-9]*.scope__;s_/cfengine3.service__' /proc/$pid/cgroup | egrep '[0-9]+:devices:/[^:]' > /dev/null; then
```

Comment:
I think **both** entries of `$pid` should be replaced with `self` here

Reply:
yeah i see i had a bad diff i go tit., will fix and re-submit